README_ru.md

```
@@ -0,0 +1,415 @@
+    </a>
+</p>
+
+<h4 align="center">
+  <a href="https://cloud.ragflow.io">Облако</a> |
+  <a href="https://ragflow.io/docs/dev/">Документация</a> |
+  <a href="https://github.com/infiniflow/ragflow/issues/12241">Дорожная карта</a> |
+  <a href="https://discord.gg/NjYzJD3GM3">Discord</a>
+</h4>
+
+<div align="center" style="margin-top:20px;margin-bottom:20px;">
+<img src="https://raw.githubusercontent.com/infiniflow/ragflow-docs/refs/heads/image/image/ragflow-octoverse.png" width="1200"/>
```

markdownlint-cli2 (0.23.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)